python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -2001,7 +2002,7 @@ def test_series_shape():
 
 def test_series_shape_empty():
     ps = pd.Series(dtype="float64")
-    cs = cudf.Series([])
+    cs = _create_cudf_series([])
```

Comment:
Rather than call this helper function to make cudf use a float64, shouldn't we alter this test to call `pd.Series([])`?

Reply:
Changed it to `Series([], dtype="float64")` and other places below. But retained the helper function calls in few places which can be used generically.